_pkgdown.yml

```diff
@@ -65,10 +65,11 @@ articles:
   - title: "More details"
     desc: >
       More information about compilation, passing in data,
-      how CmdStan ouput is written to CSV and read back into R, and
-      using CmdStanR in R Markdown documents.
+      how CmdStan ouput is written to CSV and read back into R,
+      profiling Stan programs, and using CmdStanR in R Markdown documents.
     contents:
       - cmdstanr-internals
+      - profiling
```

Comment:
When creating a new vignette you need to add the file name (without the .Rmd) to the articles section of _pkgdown.yml